### DISCUSSION TITLE
Support radio-browser.info radios list

### DISCUSSION THREAD

**Discussion by grzywek** [music-assistant/support#1100] [Feature requests and ideas]
Hi, would be nice to display radio-browser.info radios list directly in the Music Assistant.

Currently can play them via Radio Browser integration but would be nice to have all in one place.

Thanks
Dave

**Comment by yalexau** [comment]
Second this request - it would truly enable Music Assistant to be a jukebox across music sources.

**Reply by OzGav** [reply]
You asked for it…. https://github.com/orgs/music-assistant/discussions/1202

**Reply by yalexau** [reply]
Thank you, look forward to the sequent releases so I can try it out.

**Comment by Greylinux** [comment]
I just came here to see if music assistant had this exact option .

**Comment by agdamsbo** [comment]
This would be very nice. The HA implementation really lags a search function and a favorite function, so these enhancements would also really be nice.

**Comment by marcelveldt** [comment]
support for Radio Bowser (including search and favorites) has been added in the MA v2.0 beta 29, launched today.
